src/main/java/org/quiltmc/loader/impl/metadata/qmj/ModContributorImpl.java

```diff
@@ -34,6 +36,11 @@ public String name() {
 
 	@Override
 	public String role() {
-		return role;
+		return roles.get(0);
```

Comment:
This will crash if there are no roles assigned to the contributor. There should be some fallback value for this.

Reply:
I've made this return an empty string for now, open to other suggestions though.